humanguide/other-design-considerations/convergence.rst

```diff
@@ -0,0 +1,22 @@
+Convergence
+===========
+
+.. figure:: /_static/images/humanguide/41.png
+
+Convergence is one of the milestones Ubuntu Touch wants to achieve. One app available across all screen sizes and input methods, such as mobile handsets, tablets, and laptops.
```

Comment:
Maybe reword this to define Convergence sooner.

"Convergence, or having one app available across all screen sizes and input methods, is one of the main design goals of Ubuntu Touch."